setup.py

```diff
@@ -1,46 +1,41 @@
 #!/usr/bin/env python
 
-import sys
-from os.path import join, dirname
+from os.path import abspath, dirname, join
 
-sys.path.append(join(dirname(__file__), 'src'))
 from ez_setup import use_setuptools
```

Comment:
... this line requires ez_setup which we include under the src directory. We should actually remove the dependency on ez_setup.py and remove it from source.